Configuration/StandardSequences/python/GeometryConf.py

```diff
@@ -32,6 +32,7 @@
     'TrackerOnly' : 'Configuration.Geometry.GeometrySimTracker_cff,Configuration.Geometry.GeometryRecoTracker_cff',
     'HCal'        : 'Configuration.Geometry.GeometrySimHCAL_cff,Configuration.Geometry.GeometryRecoHCAL_cff',
     'Extended2023sim'   : 'Extended2023sim,Extended2023simReco',
-    'Extended2023dev'   : 'Extended2023dev,Extended2023devReco'
+    'Extended2023dev'   : 'Extended2023dev,Extended2023devReco',
+    'Extended2023LReco'   : 'Extended2023LReco,Extended2023LRecoReco'
```

Comment:
@boudoul - perhaps, the following is better as at this point the configuration files are identical:
```
'Extended2023LReco' : 'Extended2023sim,Extended2023simReco'
```